apps/e2e/mix.exs

```diff
@@ -25,6 +25,7 @@ defmodule E2E.MixProject do
 
   defp deps do
     [
+      {:alchemist, in_umbrella: true},
```

Comment:
can we bump the e2e mix version? To `0.2` since it includes alchemist now? 

(maybe this doesn't matter because we don't release the e2e test)

Reply:
Updated!